tests/Kernel/Paragraphs/ParagraphsTest.php

```diff
@@ -194,64 +188,45 @@ public function testListItem(): void {
     $this->assertCount(1, $link_element);
     $this->assertEquals('http://www.example.com/', $link_element->attr('href'));
 
-    $image_element = $crawler->filter('.ecl-list-item__primary img.ecl-image');
+    $image_element = $crawler->filter('article.ecl-card header.ecl-card__header div.ecl-card__image');
     $this->assertCount(1, $image_element);
-    $this->assertEquals(
+    $this->assertContains(
       file_url_transform_relative(file_create_url($image->getFileUri())),
-      $image_element->attr('src')
+      $image_element->attr('style')
     );
-    $this->assertEquals('Druplicon', $image_element->attr('alt'));
-
+    // @todo will be fixed on OPENEUROPA-2123.
+    // $this->assertEquals('Druplicon', $image_element->attr('alt'));
     // Change the variant to thumbnail primary.
     $paragraph->get('oe_paragraphs_variant')->setValue('thumbnail_primary');
     $paragraph->save();
 
     $html = $this->renderParagraph($paragraph);
     $crawler = new Crawler($html);
 
-    $this->assertCount(1, $crawler->filter('.ecl-list-item.ecl-list-item--thumbnail'));
-    $this->assertEquals('Item title', trim($crawler->filter('.ecl-list-item__title')->text()));
-    $this->assertEquals('Item description', trim($crawler->filter('.ecl-list-item__detail')->text()));
+    $this->assertEquals('Item title', trim($crawler->filter('article.ecl-content-item div.ecl-content-item__title')->text()));
+    $this->assertEquals('Item description', trim($crawler->filter('article.ecl-content-item div.ecl-content-item__description')->text()));
 
-    $link_element = $crawler->filter('.ecl-list-item__link');
+    $link_element = $crawler->filter('article.ecl-content-item div.ecl-content-item__title a.ecl-link');
     $this->assertCount(1, $link_element);
     $this->assertEquals('http://www.example.com/', $link_element->attr('href'));
 
-    $image_element = $crawler->filter('.ecl-list-item__primary img.ecl-image');
+    $this->assertCount(3, $crawler->filter('article.ecl-content-item > div'));
+
+    $image_element = $crawler->filter('article.ecl-content-item > div.ecl-u-d-lg-block');
     $this->assertCount(1, $image_element);
-    $this->assertEquals(
+    $this->assertContains(
       file_url_transform_relative(file_create_url($image->getFileUri())),
-      $image_element->attr('src')
+      $image_element->attr('style')
     );
-    $this->assertEquals('Druplicon', $image_element->attr('alt'));
+    $this->assertEquals('Druplicon', $image_element->attr('aria-label'));
 
-    $meta_elements = $crawler->filter('.ecl-meta__item');
-    $this->assertCount(3, $meta_elements);
-    $this->assertEquals('Meta 1', trim($meta_elements->getNode(0)->nodeValue));
-    $this->assertEquals('Meta 2', trim($meta_elements->getNode(1)->nodeValue));
-    $this->assertEquals('Meta 3', trim($meta_elements->getNode(2)->nodeValue));
-
-    // The secondary image markup should not be rendered.
-    $this->assertCount(0, $crawler->filter('.ecl-list-item__secondary img.ecl-image'));
```

Comment:
We should keep this assertion when testing the `thumbnail_primary` variant.

Reply:
I chose that we test that the image is print only once.